cpp/src/io/parquet/writer_impl.cu

```diff
@@ -99,6 +99,10 @@ struct aggregate_writer_metadata {
     }
   }
 
+#ifndef CUDF_VERSION
+#error "CUDF_VERSION is not defined"
+#endif
```

Comment:
This is in the middle of the `aggregate_writer_metadata` so the indentation here is terrible. I would recommend moving it to the top of this file, or moving it inside of the `get_metadata` below, just right before line 115.

Reply:
I wanted it close to where it's used, but it looked uglier to me in the middle of the function.  Moved it to the top just after the includes.